src/content/ui/animations/staggered-animations.md

```diff
@@ -115,26 +70,52 @@ You might notice the following characteristics:
 
 ![Diagram showing the interval specified for each motion](/assets/images/docs/ui/animations/StaggeredAnimationIntervals.png)
 
+### Watch the video
+
+To watch this example in action, watch the following video.
+
+<iframe width="560" height="315" src="{{site.yt.embed}}/0fFvnZemmh8" title="Watch this example of a staggered animation in Flutter" {{site.yt.set-short}}></iframe>
+
+This video renders the following animation of a single widget.
+It begins as a bordered blue square with slightly rounded corners.
+The square runs through changes in the following order:
+
+1. Fades in
+1. Widens
+1. Becomes taller while moving upwards
+1. Transforms into a bordered circle
+1. Changes color to orange
+
+After running forward, the animation runs in reverse.
+
+## Build a basic staggered animation
+
+This guide shows how to build a staggered animation in Flutter
+using the [basic_staggered_animation][] example.
+This example shows a series of sequential and overlapping animations
+of a single widget. Tapping the screen begins an animation
+that changes opacity, size, shape, color, and padding.
+
 To set up the animation:
 
-* Create an `AnimationController` that manages all of the
-  `Animations`.
-* Create a `Tween` for each property being animated.
-  * The `Tween` defines a range of values.
-  * The `Tween`'s `animate` method requires the
-    `parent` controller, and produces an `Animation`
-    for that property.
-* Specify the interval on the `Animation`'s `curve` property.
+1. Create an `AnimationController` that manages all of the
+   `Animations`.
+1. Create a `Tween` for each property being animated.
+   * The `Tween` defines a range of values.
+   * The `Tween`'s `animate` method requires the
+     `parent` controller, and produces an `Animation`
+     for that property.
+1. Specify the interval on the `Animation`'s `curve` property.
 
 When the controlling animation's value changes,
 the new animation's value changes, triggering the UI to update.
 
+### Tween a widget's width
```

Comment:
Rephrase; Tween is not a verb.

Reply:
In the animation business, it is: https://en.wikipedia.org/wiki/Inbetweening.

Reply:
Perhaps "Interpolate" or "Transition" would read better when a verb is needed? That would avoid conflation with "tween" as the type (or instance of that type), and potentially allow better machine translation too.

"Interpolate between a widget's widths" or "Transition between a widget's widths"

Reply:
Yes, to both of Parker's suggestions.